### DISCUSSION TITLE
CrazyFlie 2.1 on Visual Studio Code

### DISCUSSION THREAD

**Discussion by wsribunma** [bitcraze/discussions#213] [Q&A] [answered]
I am trying to run my CrazyFlie 2.1 code through Visual Studio Code. However, running the code outputs an error "ModuleNotFoundError: No module named 'cflib'" although I have already installed cflib on my Terminal using 'pip3 install cflib'.

I am not sure if there is any solution to this. I was able to run the CrazyFlie drone through IDLE on my computer but not through VS code.

**Comment by krichardsson** [comment, marked as answer]
That should work. 

If you have multiple python versions installed, you are perhaps running on a different version in VS code compared to what you have in your terminal? Then cflib is installed in a the "terminal python" and the "VS code python" will not find it. You can see the python installation that is used by VS code in the bar at the bottom of VS code.

**Reply by wsribunma** [reply]
Thank you, I just found out that my VS code was using a slightly different python version than what was working from IDLE
